src/app/core/store/effects/servables.effects.ts

```diff
@@ -36,11 +36,13 @@ export class ServablesEffects {
     exhaustMap(({ name }) =>
       this.servablesService.delete(name).pipe(
         map(() => deleteServableSuccess({ name })),
-        catchError(error => of(deleteServableFailed({ error })).pipe(
-          tap(({ error }) =>
-            this.snackbarService.show({message: error})
-          )
-        ))
+        catchError(error => {
+          let message = error.split(',').join(', ');
```

Comment:
You can use more verbose method there 
* error.replace(/,/g, ", ")
* error.replaceAll("," , ", ")
* Lodash: replace(error, ",", ", ")